apps/api/src/app/workflows-v2/usecases/placeholder-enrichment/transform-placeholder-usecase.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use a specific rule in the eslint ignore statement to ensure that other rules are ignored accidentally.‏